.go-version

```diff
@@ -1 +1 @@
-1.70.5
+1.70.6
```

Comment:
Don't forget to push a new git tag as well to kick off the release to the registry!

And thanks for the fix!